src/mrpro/operators/models/Molli.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023 Physikalisch-Technische Bundesanstalt
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+import torch
+from einops import rearrange
+
+from mrpro.operators import Operator
+
+
+class Molli(Operator):
+    def __init__(self, ti: torch.Tensor, n: torch.Tensor, rr: torch.Tensor):
+        """Parameters needed to compute t = ti + (n - 1) * rr.
+
+        Parameters
+        ----------
+        ti
+            inversion times.
+        n
+            image number within the Look-Locker experiment
+        rr
+            heartbeat interval
+
+        """
+        super().__init__()
+        self.ti = torch.nn.Parameter(ti, requires_grad=ti.requires_grad)
```

Comment:
also the MOLLI signal model only requires ti as a parameter. These are the actual inversion times of the images already taking the timing of the inversion pulse and the cardiac cycle length into account.